docs/opcodes/44_merge.mdx

```diff
@@ -0,0 +1,22 @@
+---
+fork: Merge
```

Comment:
I think this filename and references to the Merge should be called Paris.

https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md

Reply:
I totally agree with you, and initially this was the name I was using.
The caveat with using that name, was that the core library we use in our app - `@ethereumjs` uses the name `merge` to refer to the PoS transition. (See [here](https://github.com/ethereumjs/ethereumjs-monorepo/tree/a7bb5ba8284894044bbb19f91f3804e901e6d2a0/packages/common/src/hardforks))
I eventually decided to go with their nomenclature in order to avoid mixed references and naming in our app, which IMO is worse than having an inaccurate but consistent references.

Opened [an issue](https://github.com/ethereumjs/ethereumjs-monorepo/issues/2360) on their GH, I invite you to +1 it, and when they complete their renaming, we can too :)

Reply:
Got it. I +1 the issue you opened. Thanks for explaining.

Reply:
Just a short mention: we can't do a short-term update on this due to compatibility issues, so this won't go into the imminent releases. We'll need to give this an additional analysis if it's worth to add an alias (we can't just remove "Merge", again: backwards compatibility). Generally you are totally right, this is mis-named.